apps/marketing/content/design-system.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Design System
+---
+
+# Design System
+
+At Documenso we aim to be a design driven company. We believe that design is not just about how things look, but also how they work. We want to make sure that our products are easy to use and intuitive. We also want to make sure that our products are consistent and that they look and feel like they belong together.
+
+To achieve this we have created our design system which contains our primitives, token and screens alongside branding assets.
+
+We're open-sourcing our design system so that you can see how we build our products and think about design as a whole.
+
+<iframe
+  src="https://documen.so/design-system-embed"
```

Comment:
what do we want to embed here? 

we plan to publish the design system to the Figma community so anyone can remix it. the current embed links to a read-only page. it'd be best to link to the Figma community file.

[See example](https://www.figma.com/community/file/1182079839084966604/ADS%3A-Design-Tokens)

Reply:
Honestly an embed vs the figma community file is of very little difference. Jumping into a read-only will still let you create a local copy which is all the same.

This provides some interactivity on the page which is quite low-touch right now.

Reply:
> Honestly an embed vs the figma community file is of very little difference. Jumping into a read-only will still let you create a local copy which is all the same.

nope, there are *not so little* differences. 
1. when in read-only mode, you can just view the file. you don't access the data you need -- text style, spacings, measurements, and so on. 
2. you can't duplicate a Figma file or save it to your drafts. you have to invite people manually. plus, 
3. it doesn't leverage Figma as a distribution channel -- no likes, no comments, no search, no discovery.

these are significant differences. it's best to publish it to the Figma community and add a link here :)

Reply:
"when in read-only mode, you can just view the file. you don't access the data you need -- text style, spacings, measurements, and so on." => you can access those, just need to be logged in, I checkd

Reply:
- nur sure about 2, 3 is a thing
- Why wouldn't we publish after embed? Leverage both

Reply:
> Why wouldn't we publish after embed?
compound effect